content/guides/getting-started.md

```diff
@@ -0,0 +1,235 @@
+
+There are problems with managing JavaScript projects this way:
+
+- It is not immediately apparent that the script depends on an external library.
+- If a dependency is missing, or included in the wrong order, the application will not function properly.
+- If a dependency is included but not used, the browser will be forced to download unnecessary code.
+
+Let's try using webpack to manage these scripts instead...
+
+
+## Creating a Bundle
+
+First we'll tweak our directory structure slightly, separating the "source" code we write and edit (`/src`) from our "distribution" code (`/dist`) which is actually shipped to the browser:
+
+``` diff
+webpack-demo
+|- package.json
++ |- dist
++   |- index.html
+- |- index.html
+|- /src
+  |- index.js
+```
+
+To bundle the `lodash` dependency with `index.js`, we'll need to install the library locally...
+
+```bash
+npm install --save lodash
+```
+
+and then import it from our script...
+
+__src/index.js__
+
+```diff
++ import _ from 'lodash';
+
+function component () {
+// ...
+```
+
+We also need to change our update our `<script>` tags to point to the unified bundle:
+
+__dist/index.html__
+
+```diff
+ <html>
+  <head>
+    <title>webpack 2 demo</title>
+-   <script src="https://unpkg.com/lodash@4.16.6"></script>
+  </head>
+  <body>
+-   <script src="src/index.js"></script>
++   <script src="dist/bundle.js"></script>
+  </body>
+ </html>
+```
+
+In this setup, `index.js` explicitly requires `lodash` to be present, and binds it as `_` (no global scope pollution). By stating what dependencies a module needs, webpack can use this information to build a dependency graph. It then uses the graph to generate an optimized bundle where scripts will be executed in the correct order.
+
+With that said, let's run `webpack` with our script as the [entry point](/concepts/entry-points) and `bundle.js` as the [output](/concepts/output):
+
+```bash
+./node_modules/.bin/webpack src/index.js dist/bundle.js
+
+Hash: ff6c1d39b26f89b3b7bb
+Version: webpack 2.2.0
+Time: 385ms
+    Asset    Size  Chunks                    Chunk Names
+bundle.js  544 kB       0  [emitted]  [big]  main
+   [0] ./~/lodash/lodash.js 540 kB {0} [built]
+   [1] (webpack)/buildin/global.js 509 bytes {0} [built]
+   [2] (webpack)/buildin/module.js 517 bytes {0} [built]
+   [3] ./src/index.js 278 bytes {0} [built]
+```
+
+T> Output may vary. If the build is successful then you are good to go.
+
+Open `index.html` in your browser and, if everything went right, you should see the following text: 'Hello webpack'.
+
+
+## ES2015 Modules
+
+Notice the use of [ES2015 module `import`](https://developer.mozilla.org//en-US/docs/Web/JavaScript/Reference/Statements/import) (alias ES2015, *harmony*) in `src/index.js`? Although `import`/`export` statements are not supported in browsers (yet), webpack supports them and will replace those instructions with ES5 compatible wrapper code in the output. Inspect `dist/bundle.js` if you need to assure yourself.
```

Comment:
This might get a little technical for beginners, maybe we can word it a little friendlier?

Suggestion: "Although `import`/`export` statements are not supported in most browsers (yet), webpack does support them. Behind the scenes, webpack actually "transpiles" the code so that older browsers can also run it. If you inspect `dist/bundle.js`, you might be able to see how webpack does this, it's quite ingenious!"